packages/amount-selectors/src/components/AmountSelectors.js

```diff
@@ -102,7 +111,24 @@ class AmountSelectors extends React.Component {
   };
 
   render() {
-    const { disabled } = this.props;
+    const {
+      disabled,
+      max,
+      min,
+      minAmountMessage,
+      maxAmountMessage,
+      showBoundariesMessageOnlyOnFocus,
+    } = this.props;
+    const { focused, value } = this.state;
+    const showMessage =
+      !showBoundariesMessageOnlyOnFocus ||
+      (showBoundariesMessageOnlyOnFocus && focused);
```

Comment:
You don't need to check if `showBoundariesMessageOnlyOnFocus` is Truly here, you already checked it in the previous line.